Exit with non-zero code on error.

dianashk opened the pull request:
Replace postinstall script with preinstall check for dependencies.

Fixes pelias/fences-builder#17
Related to pelias/fences-builder#13

missinglink commented:
code review :+1:

Ubuntu provides the `osmfilter` and `osmconvert` tools in a package called [osmctools](http://packages.ubuntu.com/search?keywords=osmctools) which can be installed with `[sudo] apt-get install osmctools`

Unless they changed it recently `child_process.spawn` does not copy ENV vars between the parent and the child, not sure if you need that, just a heads-up :)

dianashk commented:
@missinglink thanks for the feedback

 * Added Ubuntu info to the README.
 * Added a bunch of unit tests (originally figured having tests in each of the dependencies was sufficient, but you calling it out ensured I "do the right thing" TM.
